theano/sandbox/cuda/blas.py

```diff
@@ -995,20 +995,20 @@ def c_code_helper(self, bottom, weights, top, direction, sub, height=None, width
 class GpuCorrMM(BaseGpuCorrMM):
     """GPU correlation implementation using Matrix Multiplication.
 
-    :param border_mode: currently supports "valid" only; "full" can be
-        simulated by setting `pad="full"` (at the cost of performance), or
-        by using `GpuCorrMM_gradInputs`
+    :param border_mode: the width of a border of implicit zeros to pad the
+        input with. Must be a tuple with 2 elements giving the numbers of rows
+        and columns to pad on each side, or a single integer to pad the same
+        on all sides, or a string shortcut setting the padding at runtime:
+        ``'valid'`` for ``(0, 0)`` (valid convolution, no padding), ``'full'``
+        for ``(kernel_rows - 1, kernel_columns - 1)`` (full convolution),
+        ``'half'`` for ``(kernel_rows // 2, kernel_columns // 2)`` (same
+        convolution for odd-sized kernels).
```

Comment:
thanks for the update. But I think this description isn't clear that the number given is used at the top and bottom. It is not the total split in 2. There was system with the other semantic.

I'm not sure how to make this clear, so if you don't know how to update it, I'll merge, but if you find how to clarify this, it would be great.